src/metldata/builtin_transformations/copy_content/data_transform.py

```diff
@@ -32,7 +32,7 @@ def copy_content(
     """Apply all transformation instructions."""
     return transform_data(
         data=data,
-        instructions_by_class=instructions_by_class,
+        instructions_by_class=instructions_by_class,  # type: ignore
```

Comment:
Please see my comment above related to the `Instruction` type. That should solve the type problem we suppress here.